.Lib9c.Tests/Action/Scenario/Pet/AdditionalOptionRateByFixedValueTest.cs

```diff
@@ -0,0 +1,185 @@
+        {
+            (_tableSheets, _agentAddr, _avatarAddr, _initialStateV1, _initialStateV2)
+                = InitializeUtil.InitializeStates();
+            _inventoryAddr = _avatarAddr.Derive(LegacyInventoryKey);
+            _worldInfoAddr = _avatarAddr.Derive(LegacyWorldInformationKey);
+            _recipeAddr = _avatarAddr.Derive("recipe_ids");
+        }
+
+        [Theory]
+        [InlineData(73, 10114000, 1)]
+        [InlineData(37, 10114000, 30)]
+        public void CombinationEquipmentTest(
+            int randomSeed,
+            int targetItemId,
+            int petLevel
+        )
+        {
+            var (beforeResult, afterResult) = (false, false);
+            // Get Recipe
+            var recipe = _tableSheets.EquipmentItemRecipeSheet.Values.First(
+                recipe => recipe.ResultEquipmentId == targetItemId
+            );
+            Assert.NotNull(recipe);
+
+            // Get Materials and stages
+            List<EquipmentItemSubRecipeSheet.MaterialInfo> materialList =
+                recipe.GetAllMaterials(
+                    _tableSheets.EquipmentItemSubRecipeSheetV2, CraftType.Premium
+                ).ToList();
+            var stageList = List.Empty;
+            for (var i = 0; i < recipe.UnlockStage; i++)
+            {
+                stageList = stageList.Add(i.Serialize());
+            }
+
+            var stateV2 = _initialStateV2.SetState(_recipeAddr, stageList);
+            stateV2 = CraftUtil.UnlockStage(
+                stateV2,
+                _tableSheets,
+                _worldInfoAddr,
+                recipe.UnlockStage
+            );
+
+            var subRecipe = _tableSheets.EquipmentItemSubRecipeSheetV2[recipe.SubRecipeIds![1]];
+            var (originalOption2Ratio, originalOption3Ratio, originalOption4Ratio) =
+                (subRecipe.Options[1].Ratio, subRecipe.Options[2].Ratio,
+                    subRecipe.Options[3].Ratio);
+            var (expectedOption2Ratio, expectedOption3Ratio, expectedOption4Ratio) =
+                (originalOption2Ratio, originalOption3Ratio, originalOption4Ratio);
+
+            // Get pet
+            var petRow = _tableSheets.PetOptionSheet.Values.First(
+                pet => pet.LevelOptionMap[(int)petLevel!].OptionType == PetOptionType
+            );
+            _petId = petRow.PetId;
+            stateV2 = stateV2.SetState(
+                PetState.DeriveAddress(_avatarAddr, (int)_petId),
+                new List(_petId!.Serialize(), petLevel.Serialize(), 0L.Serialize())
+            );
+            var multiplier = (100 + petRow.LevelOptionMap[(int)petLevel].OptionValue) / 100;
+            (expectedOption2Ratio, expectedOption3Ratio, expectedOption4Ratio) =
+            (
+                (int)Math.Round(originalOption2Ratio * multiplier),
+                (int)Math.Round(originalOption3Ratio * multiplier),
+                (int)Math.Round(originalOption4Ratio * multiplier)
+            );
```

Comment:
If you use Multiplier, the option type should be `AdditionalOptionRate`.
`AdditionalOptionRateByFixedValue` just adds possibility by percent point.